### PR TITLE
Task#6 - Feat: Create header section

### LINKED ISSUE #14: DevTask - Create section components
- [x] #15 
- [x] #17 
- [x] #19
- [x] #21 

After fix reponsive for all section
- [x] #32

### LINKED ISSUE #17: Task#6 - DevTask: Create header section
## OVERVIEW
- Create section components: Header
   -  header-carousel
   -  hedaer-tab
- Reponsive
- Testing
### DEMO
image

### PR DESCRIPTION
**Pull request by TheHienDuong**
- Closes #17 
## OVERVIEW
- Create header section
    - HeaderCarousel
    - HeaderTab
- Testing

## Demo
image

### CODE REVIEW

File: practice-one/src/Components/Section/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const HeaderCarousel = ({
+  childrenBlack,
+  childrenGray,
+}) => (
+  <h1 className="text-black text-6xl font-bold pt-[3rem] pb-24">
+    <span className="text-black">{childrenBlack}</span>
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Header.jsx]: what is childrenblack, nameless!

File: practice-one/src/Components/Section/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import PropTypes from 'prop-types';
+
+const HeaderCarousel = ({
+  childrenBlack,
+  childrenGray,
+}) => (
+  <h1 className="text-black text-6xl font-bold pt-[3rem] pb-24">
+    <span className="text-black">{childrenBlack}</span>
+    <br />
+    <span className="text-gray-500">{childrenGray}</span>
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Header.jsx]: The same here

File: practice-one/src/Components/Section/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  childrenBlack: PropTypes.node.isRequired,
+  childrenGray: PropTypes.node.isRequired,
+};
+
+const HeaderTab = ({
+  children,
+  onClick,
+}) => (
+  <li className="list-none hover:rounded text-gray-500">
+    <button
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Header.jsx]: this one is a `a` tag

File: practice-one/src/Components/Section/Header.jsx (hunk 1 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Header.jsx]: Yes,changed naming props

File: practice-one/src/Components/Section/Header.jsx (hunk 2 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Header.jsx]: Yes! Changed

File: practice-one/src/Components/Section/Header.jsx (hunk 3 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Header.jsx]: Yes, I changed `button` tag into `a` tag and add new handel for hover `a` tag

File: practice-one/src/Components/Section/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+// HeaderCarousel
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Header.jsx]: useless comment

File: practice-one/src/Components/Section/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    <span className="text-gray-500">{subtitle}</span>
+  </h1>
+);
+
+HeaderCarousel.propTypes = {
+  title: PropTypes.node.isRequired,
+  subtitle: PropTypes.node.isRequired,
+};
+
+// HeaderTab
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Header.jsx]: useless comment

File: practice-one/src/Components/Section/Header.jsx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+      )}
+    </li>
+  );
+};
+
+HeaderTab.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+// Header
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Header.jsx]: useless comment

File: practice-one/src/App.jsx [hunk 7]
```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 // Components
```
Comment (phattruong-agilityio) [inline practice-one/src/App.jsx:3]: Add break line

**Review by phattruong-agilityio — APPROVED** [review]

File: practice-one/src/App.jsx (hunk 7 above)
Comment (TheHienDuong) [reply practice-one/src/App.jsx:3]: Done

File: practice-one/src/Components/Section/Header.jsx (hunk 4 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Header.jsx]: Done

File: practice-one/src/Components/Section/Header.jsx (hunk 5 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Header.jsx]: Done

File: practice-one/src/Components/Section/Header.jsx (hunk 6 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Header.jsx]: Done

**Review by phattruong-agilityio — APPROVED** [review]

**Pull request merged**